README.md

```diff
@@ -883,8 +883,7 @@ Other Style Guides
     ```javascript
     // bad
     [1, 2, 3].map(number => {
-      const nextNumber = number + 1;
```

Comment:
Sorry, I don't understand then what I should save in the commit. Should I remove `return` so that the code will throw as the other `reduce` example? Or should I just close the PR?